packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -0,0 +1,345 @@
+
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as iam from '@aws-cdk/aws-iam';
+import { Construct, Duration, IResource, Resource, Stack } from '@aws-cdk/core';
+import { CfnJobDefinition } from './batch.generated';
+
+/**
+ * Properties of a job definition container
+ */
+export interface IJobDefinitionContainer {
+    /**
+     * The command that is passed to the container.
+     *
+     * If you provide a shell command as a single string, you have to quote command-line arguments.
+     *
+     * @default - CMD value built into container image.
+     */
+    readonly command?: string[];
+
+    /**
+     * The environment variables to pass to the container.
+     *
+     * @default none
+     */
+    readonly environment?: { [key: string]: string };
+
+    /**
+     * The image used to start a container.
+     */
+    readonly image: ecs.EcrImage;
+
+    /**
+     * The instance type to use for a multi-node parallel job. Currently all node groups in a
+     * multi-node parallel job must use the same instance type. This parameter is not valid
+     * for single-node container jobs.
+     */
+    readonly instanceType: ec2.InstanceType;
+
+    /**
+     * The IAM role that the container can assume for AWS permissions.
+     *
+     * @default - An IAM role will created
+     */
+    readonly jobRole?: iam.IRole;
+
+    /**
+     * Linux-specific modifications that are applied to the container, such as details for device mappings.
+     * For now, only the `devices` property is supported.
+     *
+     * @default - None will be used
+     */
+    readonly linuxParams?: ecs.LinuxParameters;
+
+    /**
+     * The hard limit (in MiB) of memory to present to the container. If your container attempts to exceed
+     * the memory specified here, the container is killed. You must specify at least 4 MiB of memory for a job.
+     *
+     * @default 4
+     */
+    readonly memoryLimitMiB?: number;
+
+    /**
+     * The mount points for data volumes in your container.
+     *
+     * @default - No mount points will be used
+     */
+    readonly mountPoints?: ecs.MountPoint[];
+
+    /**
+     * When this parameter is true, the container is given elevated privileges on the host container instance (similar to the root user).
+     * @default false
```

Comment:
Missing empty line before `@default false`.

Reply:
Got it 👍